main.tf

```diff
@@ -85,6 +85,7 @@ module "main" {
   service_account_email              = var.function_service_account_email
   timeout_s                          = var.function_timeout_s
   max_instances                      = var.function_max_instances
+  min_instances                      = var.function_min_instances
```

Comment:
We will need to add support for this in the [event-function module](https://github.com/terraform-google-modules/terraform-google-event-function) first